openlane/steps/openroad.py

```diff
@@ -782,6 +783,15 @@ class GeneratePDN(OpenROADStep):
     def get_script_path(self):
         return os.path.join(get_script_dir(), "openroad", "pdn.tcl")
 
+    def run(self, state_in: State, **kwargs) -> Tuple[ViewsUpdate, MetricsUpdate]:
+        kwargs, env = self.extract_env(kwargs)
+        if self.config["FP_PDN_CFG"] is None:
+            env["FP_PDN_CFG"] = os.path.join(
+                get_script_dir(), "openroad/common/pdn_cfg.tcl"
```

Comment:
If we're using os.path.join, may as well use three strings: "openroad" "common" "pdn_cfg.tcl"